apps/mitt-konto/src/Tracking.purs

```diff
@@ -3,10 +3,28 @@ module Tracking where
 import Effect (Effect)
 import Foreign (Foreign)
 import Prelude (Unit)
+import Data.Function.Uncurried (Fn2, Fn4, runFn2, runFn4)
 
 foreign import data Tracker :: Type
 foreign import newTracker :: Effect Tracker
 foreign import pushPageLoad :: Tracker -> Effect Unit
+foreign import reclamationEvent_ :: Fn4 Tracker Subsno DateString Claim (Effect Unit)
```

Comment:
I think it would be more appropriate to use [EffectFn](https://pursuit.purescript.org/packages/purescript-effect/2.0.1/docs/Effect.Uncurried) here, as all these are effectful functions. So for instance this would become `EffectFn4 Tracker Subsno DateString Claim Unit`